pkg/decoder/tagxl/v1/port152.go

```diff
@@ -7,16 +7,39 @@ import (
 )
 
```

Comment:
Could we perhaps have a nice inline byte / bitfield documentation above like in almost all other ports?